test/test_fixedpoint.cc

```diff
@@ -14,57 +14,324 @@
 
 #define GEMMLOWP_ENABLE_FIXEDPOINT_CONSTANTS_CHECKS
 
```

Comment:
Can you add a quick comment explaining this file's purpose? Could just be "Tests the accuracy of fixed point functions".